### PR TITLE
Reviewing streaming code and making small clean ups.

### PR DESCRIPTION
**Pull request by epugh**
Similar to PR #2530.

Typos, methods not used, return values not used, that sort of thing. Flagged by IntelliJ.

Refactoring, so no CHANGES.txt or JIRA.

### CODE REVIEW

File: solr/solrj-streaming/src/test/org/apache/solr/client/solrj/io/stream/eval/CoalesceEvaluatorTest.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*
  * Licensed to the Apache Software Foundation (ASF) under one or more
  * contributor license agreements.  See the NOTICE file distributed with
- * this work for multitional information regarding copyright ownership.
+ * this work for additional information regarding copyright ownership.
```
Comment (cpoerschke) [inline solr/solrj-streaming/src/test/org/apache/solr/client/solrj/io/stream/eval/CoalesceEvaluatorTest.java:4]: Interesting that the "file has correct license headers" checks didn't pick up on this.

Comment (epugh) [reply solr/solrj-streaming/src/test/org/apache/solr/client/solrj/io/stream/eval/CoalesceEvaluatorTest.java:4]: does it actually check the internal contents?    It looked like a set of cut n paste/ find/replace issues!

File: solr/solrj-streaming/src/test/org/apache/solr/client/solrj/io/stream/ops/ConcatOperationTest.java [hunk 2]
```diff
@@ -79,7 +79,7 @@ public void concatMultipleFields() throws Exception {
     assertNotNull(tuple.get("fieldABConcat"));
     assertEquals("bar-baz", tuple.get("fieldABConcat"));
 
-    // do the same in oposite order
+    // do the same in opposite order
```
Comment (cpoerschke) [inline solr/solrj-streaming/src/test/org/apache/solr/client/solrj/io/stream/ops/ConcatOperationTest.java:82]:
```suggestion
    // do the same in reverse order
```

**Review by cpoerschke — APPROVED** [review]

**Comment by epugh** [comment]
Realizing my other PR is stepping on this one, so going to finalize and merge this one.

**Pull request merged**